lib/matplotlib/backends/backend_wx.py

```diff
@@ -1614,10 +1614,12 @@ def press(self, event):
             if not self.retinaFix:
                 self.wxoverlay = wx.Overlay()
             else:
-                self.savedRetinaImage = self.canvas.copy_from_bbox(
-                    self.canvas.figure.gca().bbox)
-                self.zoomStartX = event.xdata
-                self.zoomStartY = event.ydata
+                if event.inaxes is not None:
+                    self.savedRetinaImage = self.canvas.copy_from_bbox(
+                        event.inaxes.bbox)
+                    self.zoomStartX = event.xdata
+                    self.zoomStartY = event.ydata
+                    self.zoomAxes = event.inaxes
```

Comment:
In this case I think it makes more senses to be consistent with the existing group of names on the 'when-in-rome' principle.

Lets leave everything as 'public' for now.